install.sh

```diff
@@ -6,6 +6,10 @@ if [[ "$(uname)" = "Linux" ]]; then
   HOMEBREW_ON_LINUX=1
 fi
 
+# Check if macOS is ARM
+if [[ "$(uname -m)" = "arm64" ]] && [[ "$(uname)" = "Darwin" ]]; then
+  HOMEBREW_APPLE_SILICON=1
+fi
```

Comment:
```suggestion
fi

```

One newline needed, not two, not none 😁